### ISSUE TITLE
Pi node is stuck

### ISSUE THREAD

**Issue by vsoft11** [pi-node/instructions#299]
![image](https://user-images.githubusercontent.com/117741830/200575560-f3a53ecf-f33f-4996-87dd-bc3b2d150f16.png)
Pi node is stuck, see attached image, anyone else facing the same issue.

**Comment by steeveee100** [comment]
Hi. Yes I did have the exact same problem. Then I went to https://t.me/PiNodechat and found the cure. Cheers.

**Comment by JohNan** [comment]
I think this could be a possible problem

```
[2022-11-22 16:56:20.462] [error] Error: Error: Cannot find channel "latest.yml" update info: HttpError: 404 Not Found
"method: GET url: https://downloads.minepi.com/node-manifests/latest.yml?noCache=1gig49njt\n\nPlease double check that your authentication token is correct. Due to security reasons actual status maybe not reported, but 404.\n"
 ```

**Comment by imacanuck2** [comment]
same issue on new computer. old one ran pi network for well over a year

**Comment by cliu2017** [comment]
Same issue on my laptop (windows 11)

**Comment by cliu2017** [comment]
Same issue for me

**Comment by cliu2017** [comment]
> Hi. Yes I did have the exact same problem. Then I went to https://t.me/PiNodechat and found the cure. Cheers.

How did you fix it?

**Comment by n37f8fib0qjcynpxyxdb** [comment]
1, Stop the docker app and Pi node.

2, Show hidden files.

Please refer to the following link below:
https://support.microsoft.com/en-us/windows/view-hidden-files-and-folders-in-windows-97fbc472-c603-9d90-91d0-1166d1d9f4b5

or find the appropriate web site for your device yourself.

3, Navigate to C:Users\yourUserName\AppData\Roaming\Pi Network
   ※(Note: The address may vary for each devices you use)

4, Make a copy of the "user-preferences.json" file and open it with Notepad, or any app which can edit the file.

5, Replace the content with the following:

{
  "windowWidth": 800,
  "windowHeight": 600,
  "uuid": "****************************************",
  "switchStatus": true,
  "dockerImage": "pinetwork/pi-node-docker:latest", 
  "containerFlavor": "stellar", ← *This is the missing part which you should paste.*
  "portsCheckPassed": false,
  "horizonStatusPreference": true,
  "stellarSecretSeed": "********************************************************"
}

Simply put, you're missing ("containerFlavor": "stellar"), 
so paste the above content at the specified location, which is after
"switchStatus":true,"dockerImage":"pinetwork/pi-node-docker:latest",

6, Reference video:https://m.youtube.com/watch?v=QXrNpntiW3M

7, After pasting and overwriting, save the file and restart Docker and Pinode.
